### ISSUE TITLE
Vite.js incompatibility

### ISSUE THREAD

**Issue by molaux** [kybarg/react-barcode-reader#42]
Hello,

I can not use your package with Vite.js since it expects package.json's `"module"` field to be a path to an es module.

It should be `"type": "commonjs"` instead I think.

```
[vite] Pre-transform error: Failed to resolve entry for package "react-barcode-reader". The package may have incorrect main/module/exports specified in its package.json.
```

[This link](https://publint.dev/react-barcode-reader@0.0.2) could be usefull.

**Comment by LucidityDesign** [comment]
Does hits work for you?
```
// @ts-expect-error
import BarcodeReader from 'react-barcode-reader/lib/index'
```

**Comment by satyam-veris** [comment]
> Does this work for you?
> 
> ```
> // @ts-expect-error
> import BarcodeReader from 'react-barcode-reader/lib/index'
> ```

I got a new error

```
Uncaught SyntaxError: The requested module '/node_modules/react-barcode-reader/lib/index.js?v=cdab873b' does not provide an export named 'default'
```

Also tried adding exports [referring to this [PR](https://github.com/kybarg/react-barcode-reader/pull/35)] in the package.json of this lib in the node_modules but getting same error

**Comment by smoelker** [comment]
Was this issue actually fixed and released? I also get this error on version 0.0.2 of the package.
